projects/storefrontlib/src/cms-components/user/login-form/login-form.module.ts

```diff
@@ -20,6 +21,7 @@ import { LoginFormComponent } from './login-form.component';
     RouterModule,
     UrlModule,
     I18nModule,
+    FeaturesConfigModule,
```

Comment:
Is it really needed here?

Reply:
seems a left over, unclear why it's here in the first place.